tests/test.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import pycurl
+import io
```

Comment:
can alphabetize the imports

Reply:
nice catch, I figured flake. would've catch that

Reply:
alphabetized imports are an extension: https://github.com/PyCQA/flake8-import-order